# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/iproute2/fix_make_install.patch
# ROCK Linux is Copyright (C) 1998 - 2004 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./Makefile.orig	2004-02-18 23:26:26.000000000 -0300
+++ ./Makefile	2004-02-18 23:47:35.000000000 -0300
@@ -65,14 +65,12 @@
 	install -m 0755 -d $(DESTDIR)$(DOCDIR)/examples
 	install -m 0755 -d $(DESTDIR)$(DOCDIR)/examples/diffserv
 	install -m 0644 README.iproute2+tc $(shell find examples -type f -maxdepth 1) $(DESTDIR)$(DOCDIR)/examples
-	@for i in $(ls -1 examples/diffserv/ | grep -v CVS); do \
-		install -m 0644 $i $(DESTDIR)$(DOCDIR)/examples/diffserv ;\
-		done
+	install -m 0644 $(shell find examples/diffserv/ -type f -maxdepth 1 | grep -v CVS) $(DESTDIR)$(DOCDIR)/examples/diffserv
 	@for i in $(SUBDIRS) doc; do $(MAKE) -C $$i install; done
-	@cd etc/iproute2; for i in $(ls -1 | grep -v CVS); do \
+	@for i in $(shell ls -1 etc/iproute2 | grep -v CVS); do \
 		if [ ! -e $(DESTDIR)$(CONFDIR)/$$i ]; then \
-			echo install -m 0644 $$i $(DESTDIR)$(CONFDIR); \
-			install -m 0644 $$i $(DESTDIR)$(CONFDIR); fi; done
+			echo install -m 0644 etc/iproute2/$$i $(DESTDIR)$(CONFDIR); \
+			install -m 0644 etc/iproute2/$$i $(DESTDIR)$(CONFDIR); fi; done
 
 clean:
 	rm -f $(shell find . -name '*~')
